CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
The CMake files should have the standard license header as well.

Reply:
Sure, I can put that there, but note that the Swift repository does not follow that convention.

Reply:
The Swift repository has a mix of this convention (https://github.com/apple/swift/blob/master/stdlib/public/core/CMakeLists.txt) and omitting them (https://github.com/apple/swift/blob/master/stdlib/public/CMakeLists.txt). I don't think there's a clear precedent.

Reply:
Oh, interesting!  Didn't notice the headers in the stdlib.  Either way, added the headers, it doesn't hurt to have them.